### PR TITLE
Prepare for 4.0.0 release

### PR DESCRIPTION
**Pull request by Sparksssj**
**Description of changes:**

Add changelog, bump version to 4.0.0.

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,32 @@
+# v4.0.0 (2024-11-20)
+
+## OS Changes
+* kernel: disable BPF preload and bpfilter helpers (#261)
+* Allow overriding the max-pods file with one from your variant (#279)
+* Update libdbus and libexpat (#270)
+* Remove acpid package (#280)
+
+
```
Comment (KCSesh) [inline CHANGELOG.md:9]: No need for extra line.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,32 @@
...
+## Orchestrator Changes
+
+  ### Kubernetes
```
Comment (KCSesh) [inline CHANGELOG.md]: Please text align this.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,32 @@
...
+* Update EKS 1.28-1.31 versions to latest (#281)
+
+
```
Comment (KCSesh) [inline CHANGELOG.md]: Please remove extra line, and match previous style.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,31 @@
+# v4.0.0 (2024-11-20)
+
+## OS Changes
+* kernel: disable BPF preload and bpfilter helpers (#261)
```
Comment (koooosh) [inline CHANGELOG.md]: Let's change this to:
`Disable BPF preload and bpfilter helpers for kernel`

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,31 @@
...
+* Allow overriding the max-pods file with one from your variant (#279)
```
Comment (koooosh) [inline CHANGELOG.md]: Nit, let's do:
`Allow overriding max-pods file with one from variant`

**Review by koooosh — APPROVED** [review]
LGTM

**Review by cbgbt — APPROVED** [review]

**Comment by Sparksssj** [comment]
Skipping CI/CD builds since it's a minor change. Updating changelog and release version.

**Pull request merged**